app/services/subscriptions/create_service.rb

```diff
@@ -50,8 +60,7 @@ def create(**args)
     private
 
     def process_create
-      return result.not_found_failure!(resource: 'customer') unless current_customer
-      return result.not_found_failure!(resource: 'plan') unless current_plan
+      return result unless valid?(customer: current_customer, plan: current_plan, subscription_date: subscription_date)
```

Comment:
this will open a discussion with @vincent-pochet I think 😂

Reply:
Done! :trollface: